src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,7 @@
+html {
+  height: 100vh;
```

Comment:
This file should be called reset.scss, normalizing is a process of normalizing the css differences between browsers, here you are resetting the styles for the purpose of your work. Also this specific rule is not a reset but rather a styling. An example of reset would be resetting all margins to 0 or setting box-sizing to border-box, but not declaring html's height

Reply:
Thank you @Radoslaw-Czerniawski for your comments. Already finished with fixing :)